### DISCUSSION TITLE
[Feature] Copy Image Coordinates on Long-press

### DISCUSSION THREAD

**Discussion by ajsnyde** [immich-app/immich#11224] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When looking at an image in map view, specifically when it shows the coords (and offers a choice to edit them), have an option to long-press them to copy the coordinates (red circled area in screencap) to the clipboard:
![image](https://github.com/user-attachments/assets/15adcf00-4586-4649-a372-ee950ef29352)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by MarioOsswald** [comment]
My workflow is: copy specific coordinates in google maps and paste them in immich for a single pic or a set of pictures.
This already works very well (one copy action from maps fills in both coordinates exactly!! - very well done :-)

But - when adding pictures attributed to the same location (which happens often over the years), it would be very useful to have a copy button inside the immich location editor to copy an existing set of coordinates from an existing picture in order to paste these coordinates into a new set of pictures (without going to google maps again or having to write down the existing coordinates otherwise).
